corehq/apps/geospatial/views.py

```diff
@@ -255,12 +255,19 @@ def _get_paginated_cases_without_gps(domain, page, limit, query):
     paginator = Paginator(case_ids, limit)
     case_ids_page = list(paginator.get_page(page))
     cases = CommCareCase.objects.get_cases(case_ids_page, domain, ordered=True)
-    case_data = [
-        {
-            'id': case_obj.case_id,
-            'name': case_obj.name,
-        } for case_obj in cases
-    ]
+    case_data = []
+    for case_obj in cases:
+        gps_location_prop = case_obj.case_json[location_prop_name]
+        gps_data = gps_location_prop.split(' ')
+        lat, lon = (gps_data[0], gps_data[1]) if len(gps_data) > 1 else ('', '')
```

Comment:
I think better to use [GeoPoint](https://github.com/dimagi/commcare-hq/blob/663ba850feae022a552202e814a579782358fe04/corehq/ex-submodules/couchforms/geopoint.py#L8)

Reply:
Switched to `GeoPoint` ([eb7cdc0](https://github.com/dimagi/commcare-hq/pull/33420/commits/eb7cdc0a1139344545ca915a3ad7746c6ec29b7d)). I also cleaned up the code a little to use a try/except block instead of a lengthy `if` statement.